### DISCUSSION TITLE
Windows quarantined streamslinks?

### DISCUSSION THREAD

**Discussion by Chris763454357** [streamlink/streamlink-twitch-gui#916] [General]
This is a weird one. 

I went to start streamlinks and the shortcut was missing. Seems this morning Microsoft quarantined "C:\Program Files\Streamlink Twitch GUI\streamlink-twitch-gui.exe" as it detected "Trojan:Win64/CryptInject!MSR"

It was version 2.1.0 downloaded from https://github.com/streamlink/streamlink-twitch-gui/releases

No other alerts for virus. All installed software on my PC is legit. No other weirdness on my PC. Only this very specific app (that can be compiled from source) got flagged. 

Ideas?

**Comment by bastimeyer** [comment]
False positive. Report it. I can't fix this.

https://github.com/streamlink/streamlink-twitch-gui/issues/914

**Reply by Chris763454357** [reply]
Ah ok, thanks for the info. I had only looked at the open issuse and the discussion forum (did not think of looking at closed).

For fun I also did a compile on this. Good instructions, the only thing I had to change was downgrading node to LTS 16 .18 (the version 18.? I had was failing).
